### PR TITLE
feat(ecocredit): add class-issuers cli command

### PR DESCRIPTION
**Pull request by aleem1314**
## Description

Closes: #1025 

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] followed the guidelines for [building modules](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules)
- [ ] included the necessary unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed 
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: x/ecocredit/client/query.go [hunk 1]
```diff
@@ -102,6 +103,47 @@ func QueryClassInfoCmd() *cobra.Command {
...
+func QueryClassIssuersCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "class-issuers [class-id]",
+		Short: "Retrieve addresses of the issuers for a credit class",
+		Long: `Retrieve addresses of the issuers for a credit class
+Args:
+	class-id: credit class id
+Example:
+	$ regen q ecocredit class-issuers C01
+	$ regen q ecocredit class-issuers C01 --pagination.limit 10
```
Comment (ryanchristo) [inline x/ecocredit/client/query.go:117]:
```suggestion
		Long: `Retrieve addresses of the issuers for a credit class.

Args:
	class-id: credit class id

Example:
	$ regen q ecocredit class-issuers C01
	$ regen q ecocredit class-issuers C01 --pagination.limit 10
```

**Review by ryanchristo — COMMENTED** [review]
This looks good to me although I think we should wait for #1026 to be merged so that we can add tests.

**Comment by ryanchristo** [comment]
Can we add a unit test for the command as it looks like we might migrate away from integration tests? #1041

**Comment by ryanchristo** [comment]
@aleem1314 it was proposed that we will treat cli tests as a last priority and as a nice to have in the backlog. I think we are ok to move forward here without the cli tests.

**Review by ryanchristo — APPROVED** [review]
This looks great. Thank you for including a test as well.

File: x/ecocredit/client/query.go [hunk 2]
```diff
@@ -102,6 +103,49 @@ func QueryClassInfoCmd() *cobra.Command {
...
+		Short: "Retrieve addresses of the issuers for a credit class",
+		Long: `Retrieve addresses of the issuers for a credit class.
+
+Args:
+	class-id: credit class id
+
+Example:
+	$ regen q ecocredit class-issuers C01
+	$ regen q ecocredit class-issuers C01 --pagination.limit 10
+		`,
```
Comment (technicallyty) [inline x/ecocredit/client/query.go:120]: can we use cobra's `Example` struct field here instead of embedding in `Long`

**Review by technicallyty — APPROVED** [review]
pre-approving, 1 nit 🙏🏻

**Pull request merged**